docker/debug_integration/mux.yml

```diff
@@ -1,21 +0,0 @@
-#
```

Comment:
@arjo129 @caguero are we actually using these scripts? I was on the fence as to whether to migrate them along or not.

Reply:
This script was a hack for back when I was working with the mbari controller in the loop. It made it easy for me to get set up with the controller

Reply:
Do you think it's useful?

Reply:
Yep, it definitely is, although a user would have to know a bit about how to navigate tmux. The top pane loads the mbari CLI and the bottom pane the simulator. The second window was for plotting vehicle behavior as seen internally by the mbari controller (we were debugging issues like sign flips for integration etc). I think the second window can be removed.

Reply:
Circling back here. On a second look, theses scripts are super specific to MBARI's codebase, so I think we should move them to the other side and make sure they are present in the public image (along with the same capabilities this tooling I'm adding has to iterate on them).